creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
-## World Differences
+## World Format Differences
 
-The most obvious difference between both versions is the world format. Bedrock Edition uses the LevelDB format for world storage while Java Edition uses the Anvil format. Due to this, most third-party tools created for Java Edition world editing will not work on Bedrock Edition.
+One major technical difference between both versions is the world format. Bedrock Edition uses Google's [LevelDB format](https://en.wikipedia.org/wiki/LevelDB) for world storage, while Java Edition uses a custom [Anvil format](https://wiki.vg/Map_Format). Due to this, most third-party tools created for world editing in a specific edition will not work on the other.
 
-The two versions also use a fairly different block format. Java Edition has flattened its block format using a unique string for each individual block and storing the state of that block separately. Similarly, Bedrock Edition has moved to a string-based system with block states, but have kept some blocks grouped together defined by data value. Basically, this means that blocks are named differently between the versions. In Bedrock Edition, granite would be `stone 1` whereas on Java Edition it’s simply `granite`.
+Many blocks and items are also named differently between the two versions. In 1.13, Java Edition underwent an internal [flattening](https://minecraft.fandom.com/wiki/Java_Edition_1.13/Flattening), removing the concept of "damage values" so that each block has its own name. Bedrock has not made this change, and so still has the pre-1.13 block names. For example, granite is `granite` in Java Edition, and `stone:1` in Bedrock Edition.
 
-## Redstone and Command Differences
+## Command Differences
 
-The structure and implementation of commands between the two versions have diverged as well. Bedrock Edition’s command structure is similar to the system used in versions of Java Edition prior to 1.13. It also forgoes raw JSON strings inside commands for a component-based system. Instead of using long complex JSON strings to customize entities, you can summon an entity with an event to fire, and also name it in a single command.
+The set of commands between the two versions has diverged as well. Java Edition [overhauled many commands](https://minecraft.fandom.com/wiki/Java_Edition_1.13#Command_format_2) in 1.13, while Bedrock made no such changes.
+
+### NBT
+Bedrock notably lacks the ability to specify NBT in any command. This is used in Java Edition to create special items and blocks, summon mobs with specific properties, check for complex conditions, and much more. In some cases, workarounds to achieve similar effects are possible.
+
+For example, compare the syntax of `/summon`:
 
```

Comment:
I don't know why this paragraph referred to NBT as JSON, in my opinion NBT is important enough to deserve its own section explicitly stating that Bedrock does not have it